middleware.go

```diff
@@ -10,13 +10,13 @@ import (
 	"go.opentelemetry.io/otel"
 	"go.opentelemetry.io/otel/propagation"
 
-	otelcontrib "go.opentelemetry.io/contrib"
-	semconv "go.opentelemetry.io/otel/semconv/v1.4.0"
+	semconv "go.opentelemetry.io/otel/semconv/v1.12.0"
 	oteltrace "go.opentelemetry.io/otel/trace"
 )
 
 const (
-	tracerName = "github.com/riandyrn/otelchi"
+	tracerName             = "github.com/riandyrn/otelchi"
+	instrumentationVersion = "0.6.0" // TODO: we need to find a way to automate this later
```

Comment:
We may add new file `version.go` that return version string like other instrumentation library. wdyt? @riandyrn

Reply:
Yes, but they are updating the version string using a pre-release script, @ilhamsyahids.

I still don't know how to adapt this flow into our workflow. This is why I decided that we will keep things simple by using a constant in the main for now. Because it simply doesn't make any difference with putting the constant on `version.go` for now.

Maybe you are interested in doing research in this area?

Reply:
Did you mean workflow like auto-update the version whenever we release? I ever seen it but I think it require some tweaking in github workflows or using tools, lets keep it as todo.

The reason putting on the `version.go` is to make exportable so that other developer can play around it like checking correctness of the library version. Well we can create in the `middleware.go` but I think its not related